code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -112,15 +125,17 @@
 	ranged = TRUE
 	harm_intent_damage = 5
 	obj_damage = 60
-	melee_damage_lower = 25
-	melee_damage_upper = 25
+	melee_damage_lower = 20
+	melee_damage_upper = 20
+	minbodytemp = 100
 	combat_mode = TRUE
 	del_on_death = TRUE
 	deathmessage = "collapses into bits of plant matter."
 	attacked_sound = 'sound/creatures/venus_trap_hurt.ogg'
 	deathsound = 'sound/creatures/venus_trap_death.ogg'
 	attack_sound = 'sound/creatures/venus_trap_hit.ogg'
-	unsuitable_heat_damage = 5 //note that venus human traps do not take cold damage, only heat damage- this is because space vines can cause hull breaches
+	unsuitable_heat_damage = 5 // note that venus human traps do not take cold damage, only heat damage- this is because space vines can cause hull breaches
```

Comment:
Please change this comment instead of commenting on the comment on the line below.